packages/devtools_app/lib/src/theme.dart

```diff
@@ -148,6 +149,7 @@ const actionsIconSize = 20.0;
 
 const defaultSpacing = 16.0;
 const denseSpacing = 8.0;
+const denseModeDenseSpacing = 0.0;
```

Comment:
do we really want 0.0? what about 2.0 so that elements aren't touching?

Reply:
This is only used for changing height of the expand/collapse buttons and the space in between them. Changing to 2.0 looks fine though.